ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/auth/TestAuditPasswordAuthenticator.java

```diff
@@ -118,8 +119,8 @@ public void testLogOnRuntimeException()
         assertThatExceptionOfType(RuntimeException.class)
         .isThrownBy(negotiator::getAuthenticatedUser);
 
-        verify(mockAdapter, times(1)).auditAuth(eq("username"), eq(Status.ATTEMPT), longThat(isCloseToNow()));
-        verify(mockAdapter, times(1)).auditAuth(eq("username"), eq(Status.FAILED), longThat(isCloseToNow()));
+        verify(mockAdapter, times(1)).auditAuth(eq("username"), eq(Status.ATTEMPT), longThat(isCloseToNow()), eq(Optional.empty()));
```

Comment:
times(1) is the default value for the verify (overloaded) method, so it could be left out

Reply:
I think it's more explicit to show what we mean. I think it's also done in several places already